pizza_cutter/des_pizza_cutter/_se_image.py

```diff
@@ -248,6 +253,28 @@ def set_slice(self, x_start, y_start, box_size):
         self.noise = nse[
             y_start:y_start+box_size, x_start:x_start+box_size].copy()
 
+    def _set_tape_bump_mask(self, bmask):
+        """
+        set the tape bump flag on the input bmask
+
+        Parameters
+        ----------
+        bmask: array
+            This must be the original array before trimming
+
+        Effects
+        ------------
+        The TAPEBUMP bit is set
+        """
+
+        ccdnum = self.source_info['ccdnum']
+        bumps = TAPE_BUMPS[ccdnum]
+        for bump in bumps:
+            bmask[
+                bump['row1']:bump['row2']+1,
+                bump['col1']:bump['col2']+1,
+            ] |= bump['flag']
```

Comment:
We should unset the SUSPECT flag here as well. This will make Y3 consistent with Y5.